ShenkinStore/Models/Transaction.cs

```diff
@@ -1,11 +1,21 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
 using System.Linq;
 using System.Threading.Tasks;
 
 namespace ShenkinStore.Models
 {
     public class Transaction
     {
+        public int TransactionId { get; set; } // primary key
```

Comment:
Please add validations for all the fields

Reply:
**Related to the upper comment**